<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ACMIPlot3D</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #999999;
	color: #000000;
	font-size: 11px;
}

QMenuBar::item {
    background: transparent;
}
QMenuBar::item:selected, QMenu::item:selected, QComboBox::hover {
    color: #000000;
    background: #ffaa00;
}

QComboBox {
	border: 1px solid #454545;
	min-height: 20px;
	padding-left: 4px;
	selection-background-color: #aaaaaa;
}
QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px;
	border-left-width: 0px;
	border-left-color: #454545;
	border-left-style: solid;
}
QComboBox::down-arrow, QScrollBar::down-arrow {
	width: 15px;
	height: 15px;
	image: url(:/icon_down.png);
}
QScrollBar::up-arrow {
	width: 15px;
	height: 15px;
	image: url(:/icon_up.png);
}

QSlider::groove:horizontal {
	border: 1px solid #454545;
	height: 5px;
	margin: 4px 0px 4px 0px;
}
QSlider::handle {
	border: 1px solid #454545;
	background: #454545;
}
QSlider::handle:horizontal {
	width: 5px;
	margin: -4px 0px -4px 0px;
}

QToolButton {
	border: 1px solid #eeeeee;
	border-radius: 1px;
	padding: 0px;
	margin: 0px;
	outline: 0px;
}
QToolButton::hover {
	border: 1px solid #454545;
}
QToolButton::pressed {
	background-color: #aaaaaa;
}

QScrollBar {
	border: 1px solid #454545;
}
QScrollBar:horizontal {
	height: 15px;
	margin: 0px 16px 0 16px;
}
QScrollBar::handle {
	min-height: 10px;
	min-width: 10px;
	border: 1px solid #454545;
	background-color: #454545;
}
QScrollBar::add-line:horizontal {
	border: 1px solid #454545;
	border-radius: 1px;
	width: 14px;
	subcontrol-position: right;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
	border: 1px solid #454545;
	border-radius: 1px;
	width: 14px;
	subcontrol-position: left;
	subcontrol-origin: margin;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
	background: none;
}
QScrollBar:vertical {
	width: 15px;
	margin: 16px 0 16px 0;
}
QScrollBar::add-line:vertical {
      border: 1px solid #454545;
      height: 14px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
      border: 1px solid #454545;
      height: 14px;
      subcontrol-position: top;
      subcontrol-origin: margin;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="GLWidget" name="openGLWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionInsertAxis">
   <property name="text">
    <string>Insert &amp;Axis</string>
   </property>
  </action>
  <action name="actionInsertTool">
   <property name="text">
    <string>Insert &amp;Tool</string>
   </property>
  </action>
  <action name="actionInsertDexel">
   <property name="text">
    <string>Insert De&amp;xel</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="text">
    <string>&amp;Delete Item</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SelectionChanged(QModelIndex)</slot>
  <slot>InsertAxis()</slot>
  <slot>InsertTool()</slot>
  <slot>DeleteItem()</slot>
  <slot>ReadFile()</slot>
  <slot>InsertDexel()</slot>
  <slot>About()</slot>
  <slot>SliderChanged(int)</slot>
 </slots>
</ui>
